### PR TITLE
Add ignoring of TimeoutException also to _wait_until

### PR DESCRIPTION
**Pull request by ljelinkova**
This is a follow-up patch to https://github.com/oVirt/ovirt-system-tests/pull/154. I am adding the comments from that patch as a separate comments to this patch.

### CODE REVIEW

**Comment by ljelinkova** [comment]
_Don't we want the WebDriverException on that ignored_exceptions too?_ (https://github.com/oVirt/ovirt-system-tests/pull/154#discussion_r877032210)

**Comment by ljelinkova** [comment]
I'd rather use f-strings throughout the code - they're much more elegant (https://github.com/oVirt/ovirt-system-tests/pull/154#discussion_r877034298)

**Comment by ljelinkova** [comment]
Let's use PEP8 conventions, so should_run_again (https://github.com/oVirt/ovirt-system-tests/pull/154#discussion_r877034958)

**Comment by ljelinkova** [comment]
Can other situations than java.util.concurrent.TimeoutException indeed cause a python TimeoutException to be raised? (https://github.com/oVirt/ovirt-system-tests/pull/154#discussion_r877039046)

File: ost_utils/selenium/navigation/driver.py [hunk 1]
```diff
@@ -155,7 +157,7 @@ def __call__(self, driver):
             raise e
         except Exception as e:
             LOGGER.exception(
-                '!!!ConditionClass failed with ' + e.__class__.__name__ + ' at retry number ' + str(self.retry)
+                f'!!! ConditionClass failed with {e.__class__.__name__} at retry number {str(self.retry)}'
```
Comment (tinez) [inline ost_utils/selenium/navigation/driver.py:160]: We don't need the `str()` conversions around `self.retry`

**Pull request merged**